src/core/sys/posix/sys/ioctl.d

```diff
@@ -323,13 +323,7 @@ else version (OSX)
     import core.sys.posix.termios; // termios
     import core.sys.posix.sys.time; // timeval
 
-    struct winsize
```

Comment:
I will move winsize back into ioctl.d. Even though I decided to follow the OSX header file model (and I will for the remaining things if there are no objections), I realized there may be existing D programs that include only ioctl.d and would have expected winsize to be present in ioctl.d itself (i.e for backwards compatibility).